viewlog.sh

```diff
@@ -0,0 +1,3 @@
+UUID=$(uuidgen)
+URL=$(curl -s "http://${1}/rest/events/subscribe?id=${UUID}&log")
+curl -s -N "http://${1}${URL}?id=${UUID}" | sed -u -n '/event: logger/{n;p;}' | cut -c 7-
```

Comment:
This is fantastic!

Reply:
> This is fantastic!

Thought you might like this.   I tested on Ubuntu ant MacOS.